src/ae_epoll.c

```diff
@@ -110,8 +110,15 @@ static int aeApiPoll(aeEventLoop *eventLoop, struct timeval *tvp) {
     aeApiState *state = eventLoop->apidata;
     int retval, numevents = 0;
 
+    #ifdef HAVE_EPOLL_PWAIT2
```

Comment:
I believe it's more common to not indent the preprocessor commands. (They are only indented within nested preprocessor scopes, like inside another `#ifdef`-`#endif` block.)

```suggestion
#ifdef HAVE_EPOLL_PWAIT2
```